alembic/versions/581774928d27_photometry_magref.py

```diff
@@ -0,0 +1,34 @@
+"""photometry magref
+
+Revision ID: 581774928d27
+Revises: 1a43ee08a734
+Create Date: 2022-03-30 15:14:59.284519
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '581774928d27'
+down_revision = '1a43ee08a734'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('photometry', sa.Column('magref', sa.Float(), nullable=True))
+    op.add_column('photometry', sa.Column('magref_unc', sa.Float(), nullable=True))
```

Comment:
I would have thought you'd need to add ref_flux and ref_fluexerr to the data model, not mag. Mag appears to be a derived property. What am I missing?

Reply:
I think this is the old migration script. I thought I had removed this but for some reason it is still on the repo. I'll remove it now.